modules/openshift-developer-cli-sample-applications.adoc

```diff
@@ -0,0 +1,114 @@
+python      openshift     2.7,3.3,3.4,3.5,3.6,latest
+ruby        openshift     2.0,2.2,2.3,2.4,latest
+wildfly     openshift     10.0,10.1,8.1,9.0,latest
+----
+
+== Examples from Git repositories
+
+=== httpd
+
+This example helps build and serve static content via httpd on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/sclorg/httpd-container/blob/master/2.4/root/usr/share/container-scripts/httpd/README.md[here].
+
+----
+$ odo create httpd --git https://github.com/openshift/httpd-ex.git
+----
+
+=== java
+
+This example helps build and run fat JAR Java applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/fabric8io-images/s2i/blob/master/README.md[here].
+
+----
+$ odo create java --git https://github.com/spring-projects/spring-petclinic.git
+----
+
+=== nodejs
+
+Build and run Node.js applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/sclorg/s2i-nodejs-container/blob/master/8/README.md[here].
+
+----
+$ odo create nodejs --git https://github.com/openshift/nodejs-ex.git
+----
+
+=== perl
+
+This example helps build and run Perl applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/sclorg/s2i-perl-container/blob/master/5.26/README.md[here].
+
+----
+$ odo create perl --git https://github.com/openshift/dancer-ex.git
+----
+
+=== php
+
+This example helps build and run PHP applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/sclorg/s2i-php-container/blob/master/7.1/README.md[here].
+
+----
+$ odo create php --git https://github.com/openshift/cakephp-ex.git
+----
+
+=== python
+
+This example helps build and run Python applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/sclorg/s2i-python-container/blob/master/3.6/README.md[here].
+
+----
+$ odo create python --git https://github.com/openshift/django-ex.git
+----
+
+=== ruby
+
+This example helps build and run Ruby applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/sclorg/s2i-ruby-container/blob/master/2.5/README.md[here].
+
+----
+$ odo create ruby --git https://github.com/openshift/ruby-ex.git
+----
+
+=== wildfly
+
+This example helps build and run WildFly applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/wildfly/wildfly-s2i/blob/master/README.md[here].
+
+----
+$ odo create wildfly --git https://github.com/openshift/openshift-jee-sample.git
+----
+
+Binary example
```

Comment:
I don't know what this syntax is or this is supposed to be, a heading? It just renders like a heading at the same level as the other examples and looks like it's missing content.

Need to rethink how to organize this. Maybe you could put both the first java example and this binary java example under the same heading and distinguish, and same w/ WildFly? Or maybe make separate headings of "Examples from Git repositories" of one for from source and the other from binaries.